portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -17,20 +17,20 @@
 /**
  * Fetches data from servlet and sets it in the comments section of portfolio.
  * Called whenever comments section is loaded.
- * TODO: Modify to handle dynamic data.
  */
 async function getData() {
   const response = await fetch('/data');
   const jsonData = await response.json();
   
   const commentsListElement = document.getElementById('comments-container');
   commentsListElement.innerHTML = '';
-  for(comment of jsonData){
-    commentsListElement.appendChild(createListElement(comment));
+  for (comment of jsonData) {
+    commentsListElement.appendChild(createListElement(comment.commentText + " -" + comment.name));
```

Comment:
Can you make the " -" into a const variable?

Reply:
See above